### PR TITLE
WSTEAMA 886 Launch Simorgh route persian/afghanistan service to homepage on live environment

### PR DESCRIPTION
**Pull request by holchris**
Resolves JIRA [886](https://jira.dev.bbc.co.uk/browse/WSTEAMA-886)

Overall changes
======
- Launch persian/afghanistan topic page to live 

Code changes
======
 
Testing
======
Removes unnecessary cypress test for persian/afghanistan idxPage

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/routes/utils/regex/utils/index.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ export const getIdxPageRegex = () => {
   let idxPageRegex = idxRegex;
 
   if (isLive()) {
-    idxPageRegex = `${idxRegex}|persian/afghanistan`;
+    idxPageRegex = `${idxRegex}`;
   }
 
   return `/:idx(${idxPageRegex}):amp(${ampRegex})?`;
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/utils/index.js]: We can just use the `idxRegex` value as declared above - no need to use a separate variable

```suggestion

  return `/:idx(${idxRegex}):amp(${ampRegex})?`;
```

File: src/app/routes/topic/index.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import { TopicPage } from '#pages';
 import { topicPath } from '#app/routes/utils/regex';
 import { TOPIC_PAGE } from '#app/routes/utils/pageTypes';
-import isLive from '#app/lib/utilities/isLive';
 import getInitialData from './getInitialData';
 
 export default {
-  path: [topicPath, !isLive() && '/persian/afghanistan'],
+  path: [topicPath && '/persian/afghanistan'],
```
Comment (karinathomasbbc) [inline src/app/routes/topic/index.js]: Ah... this won't work 😿 

The path needs to be an array of values, containing the regex for all topics (`topicPath`) as well as the "regex" (but really it's just hardcoded) for the /persian/afghanistan page
 
```suggestion
  path: [topicPath, '/persian/afghanistan'],
```

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]
See comments ☝️

**Review by shayneahchoon — APPROVED** [review]

File: src/app/routes/utils/regex/utils/index.js [hunk 3]
```diff
@@ -47,9 +47,12 @@ const homePageServices = [
   'igbo',
   'indonesia',
   'kyrgyz',
+  'korean',
   'marathi',
   'nepali',
+  'pashto',
   'persian',
+  'persian/afghanistan',
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/utils/index.js]: This is not required here - /persian/afghanistan is not a homepage, but a topic page. The changes to src/app/routes/topic/index.js and to `getIdxPageRegex` below are the code to launch /persian/afghanistan to live. We only need to include korean and pashto in the list of homepage services to migrate here.

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

**Review by karinathomasbbc — APPROVED** [review]
Nice!

**Review by alex-magana — APPROVED** [review]
LGTM.

**Pull request merged**